app/assets/json/translations/react/es.json

```diff
@@ -1222,14 +1231,18 @@
   "lottery.lotteryNumberNotValid": "Ingrese un número de sorteo válido.",
   "lottery.lotteryResults": "Resultados del Sorteo",
   "lottery.lotteryResultsPdfLink": "Descargue los Resultados del Sorteo en Formato PDF",
+  "lottery.nextStepsLearnMore": "Obtenga más información acerca de qué esperar después del sorteo.",
   "lottery.nextStepsP1": "El agente se comunicará con los solicitantes según el orden de preferencia hasta que se hayan ocupado todas las vacantes.",
   "lottery.nextStepsP2": "Si su solicitud sale elegida en el sorteo, tenga en cuenta que deberá completar una solicitud más detallada y proporcionar la documentación de respaldo requerida.",
   "lottery.nextStepsP3": "Si se ocupan todas las vacantes para una preferencia, usted seguirá siendo elegible para las unidades de su preferencia siguiente.",
   "lottery.nextStepsTitle": "¿Qué sucede después?",
   "lottery.numberApplicantsQualifiedForGeneralPool": "Hay %{number} solicitantes en esta lista. Tenga en cuenta que los titulares de preferencias se considerarán primero para todas las unidades.",
   "lottery.numberApplicantsQualifiedForPreference": "Hay %{number} solicitantes en esta lista.",
+  "lottery.numberApplicantsQualifiedForPreference.nonVeteran": "%{number} otros solicitantes %{plural} en esta lista",
+  "lottery.numberApplicantsQualifiedForPreference.veteran": "%{number} Veteranos%{plural} en esta lista",
```

Comment:
This string does not have a singular translation? This should be "1 veterano en esta lista"
<img width="193" alt="Screenshot 2024-06-17 at 9 07 29 AM" src="https://github.com/SFDigitalServices/sf-dahlia-web/assets/36685920/83f82903-fdca-4820-9718-0f3dae88f51d">

Reply:
Our translation system doesn't properly support pluralization. I discussed with Bridget and we decided that any pluralization attempts in this translation batch will inevitably lead to some small translation issues, but we can ignore them for now and fix them once this ticket is completed (which is blocked by a bloom UIC ticket): https://sfgovdt.jira.com/browse/DAH-2276?focusedCommentId=79363